contrib/inventory/ec2.py

```diff
@@ -721,6 +728,51 @@ def get_elasticache_replication_groups_by_region(self, region):
         for replication_group in replication_groups:
             self.add_elasticache_replication_group(replication_group, region)
 
+    def get_autoscaling_instances_by_region(self, region):
+        ''' Makes an AWS API call to the list of Autoscale instances
+         in a particular region with tags of their Autoscale group
+         if the "propagate_at_launch" is set '''
+
+        try:
+            conn_autoscale = self.connect_to_aws(autoscale, region)
+            conn_ec2 = self.connect_to_aws(ec2, region)
+            if conn_autoscale and conn_ec2:
+                reservations = []
+                autoscale_groups = conn_autoscale.get_all_groups()
+                for autoscale_group in autoscale_groups:
+                    if autoscale_group.name:
+                        tags = {}
+                        for tag in autoscale_group.tags:
+                            if tag.propagate_at_launch:
+                                tags[tag.key] = tag.value
+
+                        for autoscale_group_instance in autoscale_group.instances:
+
+                            if self.ec2_instance_filters:
+                                for filter_key, filter_values in self.ec2_instance_filters.items():
+                                    reservations.extend(conn_ec2.get_all_instances(
+                                        filters = {
+                                            filter_key : filter_values,
+                                            'instance-id': [ autoscale_group_instance.instance_id ]
+                                        }
+                                    ))
+                            else:
+                                reservations = conn_ec2.get_all_instances(
+                                    filters = { 'instance-id': [ autoscale_group_instance.instance_id ] })
+
+                            for reservation in reservations:
+                                for reservation_instance in reservation.instances:
+                                  reservation_instance.tags = tags
+                                  self.add_instance(reservation_instance, region)
```

Comment:
CI failure due to PEP 8 indentation issue:

```
2017-02-17 09:19:55 ERROR: PEP 8: contrib/inventory/ec2.py:792:35: E111 indentation is not a multiple of four (current)
2017-02-17 09:19:55 ERROR: PEP 8: contrib/inventory/ec2.py:793:35: E111 indentation is not a multiple of four (current)
```

You can run this test locally with `make pep8`.